src/components/Avatar/AvatarCard.test-int.js

```diff
@@ -0,0 +1,35 @@
+/* global Feature, Scenario  */
+
+const KIND = 'Avatar.AvatarCard';
+
+Feature('AvatarCard');
+
+Scenario('Storybook Documentation', (I) => {
+    I.searchWithinIframe(KIND, 'Overview', () => {
+        I.see('Avatar: AvatarCard');
+        I.see('Overview', 'h2');
+        I.see('Example', 'h3');
+        I.see('Source Code', 'h3');
+        I.seeElement('code');
+        I.see('Props', 'h3');
+        I.seeElement('table');
+        I.see('property');
+        I.see('value');
+        I.see('required');
+        I.see('default');
+        I.seeNumberOfElements('td', 6 * 4);
+    });
+});
+
+Scenario('AvatarCard', (I) => {
+    I.searchWithinIframe(KIND, 'Overview', () => {
+        I.seeElement('.uir-avatar-card');
+        I.seeElement('.uir-avatar');
+        I.seeElement('.uir-avatar-card-name');
+        I.see('David Smith');
```

Comment:
Might be better to condense lines 28 and 29 into:

I.see('David Smith', '.uir-avatar-card-name');
Then you've verified more, by checking that David Smith is within the right element, and reduced execution time (albeit by a fragment of a second)

Reply:
Nice!